content/_components/shopify-admin-v2/index.md

```diff
@@ -6,28 +6,28 @@ description: Shopify admin Component is designed to connect to Shopify GraphQL A
 icon: shopify-admin-v2.png
 icontext: Shopify Admin v2 component
 category: shopify-admin-v2
-updatedDate: 2023-02-28
-ComponentVersion: 2.2.0
+updatedDate: 2024-01-18
+ComponentVersion: 2.2.1
 ---
 
 ## Description
 
-Shopify admin Component is designed to connect to Shopify [GraphQL](https://shopify.dev/api/admin-graphql) Admin API
+Shopify admin Component is designed to connect to Shopify [GraphQL](https://shopify.dev/api/admin-graphql) Admin API.
 
-Tested with API version: `2023-01`
+Tested with API version: `2023-01`.
 
 ## Credentials
 
 To use this component you need to create a custom App:
-1. Go to your store [Admin panel](https://accounts.shopify.com/store-login)
-2. Open `App and sales channel settings`
-3. Select `Develop apps for your store`
-4. Then `Create an app` button
-5. Provide any app name and press `Create app`
-6. Follow to `Configure Admin API scopes`
-7. Select access scopes - Objects, that component will have access to and press `Save`
-8. Now you be able to install this app by pressing `Install app` button in `API credentials` section or in right upper corner
-9. Finally you can get `Admin API access token` by selecting `Reveal token once`. It will be needed in component credentials configuration
+1. Go to your store [Admin panel](https://accounts.shopify.com/store-login).
+2. Open `App and sales channel settings`.
+3. Select `Develop apps for your store`.
+4. Then `Create an app` button.
+5. Provide any app name and press `Create app`.
+6. Follow to `Configure Admin API scopes`.
+7. Select access scopes - Objects, that component will have access to and press `Save`.
+8. Now you be able to install this app by pressing `Install app` button in `API credentials` section or in right upper corner.
```

Comment:
8. Now you be able to install this app by pressing `Install app` button in `API credentials` section or in the right upper corner.